.github/workflows/manylinux/entrypoint.sh

```diff
@@ -34,11 +38,16 @@ cd $ONNX_PATH
 
 # Compile wheels
 # Need to be updated if there is a new Python Version
-declare -A python_map=( ["3.6"]="cp36-cp36m" ["3.7"]="cp37-cp37m" ["3.8"]="cp38-cp38" ["3.9"]="cp39-cp39")
-declare -A python_include=( ["3.6"]="3.6m" ["3.7"]="3.7m" ["3.8"]="3.8" ["3.9"]="3.9")
-PY_VER=${python_map[$PY_VERSION]}
-PIP_COMMAND="/opt/python/${PY_VER}/bin/pip install --no-cache-dir"
-PYTHON_COMAND="/opt/python/"${PY_VER}"/bin/python"
+if [ `uname -m` == 'aarch64' ]; then
+ PIP_COMMAND="$PY_VERSION -m pip install --no-cache-dir"
+ PYTHON_COMAND="$PY_VERSION"
+else
+ declare -A python_map=( ["3.6"]="cp36-cp36m" ["3.7"]="cp37-cp37m" ["3.8"]="cp38-cp38" ["3.9"]="cp39-cp39")
+ declare -A python_include=( ["3.6"]="3.6m" ["3.7"]="3.7m" ["3.8"]="3.8" ["3.9"]="3.9")
+ PY_VER=${python_map[$PY_VERSION]}
+ PIP_COMMAND="/opt/python/${PY_VER}/bin/pip install --no-cache-dir"
+ PYTHON_COMAND="/opt/python/"${PY_VER}"/bin/python"
```

Comment:
L43, L49
nit: this should be my typo: PYTHON_COMMAND

Reply:
Corrected.

Reply:
Also L66 and L68 for PIP_COMMAND. Thank you!